sale_margin_percentage/models/sale_order_line.py

```diff
@@ -0,0 +1,30 @@
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
```

Comment:
Could you split by model on a separate commit, please?

Reply:
Done

Reply:
Commit message: it's -> its